src/hubverse_infrastructure/hubs/iam.py

```diff
@@ -93,20 +94,55 @@ def create_bucket_write_policy(hub_name: str):
     return bucket_write_policy
 
 
-def attach_bucket_write_policy(hub_name: str, github_role, bucket_write_policy):
+def attach_bucket_write_policy(resource_name: str, role: aws.iam.Role, bucket_write_policy: aws.iam.Policy):
     """Attach the S3 write policy to the role that Github Actions assumes."""
 
     # Update the role we created for Github Actions by attaching the
     # policy that allows writes to the hub's S3 bucket
-    aws.iam.RolePolicyAttachment(resource_name=hub_name, role=github_role.name, policy_arn=bucket_write_policy.id)
```

Comment:
renamed "hub_name" to "resource_name" to make it clear that we're attaching the policy to an AWS resource (which just happens to have the same name as the hub, but we could decide to change that convention later)